src/coredump/meson.build

```diff
@@ -7,8 +7,10 @@ systemd_coredump_sources = files('''
 '''.split())
 
 if conf.get('HAVE_ELFUTILS') == 1
-        systemd_coredump_sources += files(['stacktrace.c',
-                                           'stacktrace.h'])
+        systemd_coredump_sources += files('''
+                stacktrace.c
+                stacktrace.h
+        '''.split())
```

Comment:
So... it was I who introduced this style with `''' '''.split()`, but in hindsight, maybe it wasn't the best idea. I wonder if we should just convert everything to `files()` with quoted names...

Reply:
OK. Newly introduced `''' '''.split()` are dropped.